server/server/src/discovery.rs

```diff
@@ -0,0 +1,66 @@
+use std::{
+    fmt::Display,
```

Comment:
these changes were taken from discovery PR (should be in soon). Was needed to tell front end that is served by back end, where the server is, as per pr description, need to change this